dataset-processor-python/data_loader.py

```diff
@@ -31,25 +43,186 @@ class DatasetWriter(DataLoader):
 
     def __init__(self, path, dataset_name):
         super().__init__(path, dataset_name)
+        self.cell_feature_analysis_file = []
+        self.feature_defs_file = []
+        self.dataset_file = {}
```

Comment:
It looks like these first three should not have `_file` in the name, since they are not files but the actual data..?